acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/SignTransactionAcceptanceTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.signing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.ethsigner.tests.dsl.Accounts.RICH_BENEFACTOR;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.GAS_PRICE;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.INTRINSIC_GAS;
+
+import tech.pegasys.ethsigner.tests.AcceptanceTestBase;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+import org.assertj.core.data.Offset;
+import org.junit.Test;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.utils.Convert;
+import org.web3j.utils.Convert.Unit;
+
+public class SignTransactionAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  public void valueTransfer() throws IOException {
+    final String recipient = "0x1b00ba00ca00bb00aa00bc00be00ac00ca00da00";
+    final BigInteger transferAmountWei = Convert.toWei("15.5", Unit.ETHER).toBigIntegerExact();
+    final BigInteger startBalance = ethNode().accounts().balance(recipient);
+
+    final Transaction transaction =
+        Transaction.createEtherTransaction(
+            RICH_BENEFACTOR.address(),
+            RICH_BENEFACTOR.getNextNonceAndIncrement(),
+            GAS_PRICE,
+            INTRINSIC_GAS,
+            recipient,
+            transferAmountWei);
+
+    final String hash = ethSigner().transactions().submit(transaction);
```

Comment:
nit: wrt other comment - other ways of writing this:
transaction.submit(ethSigner)
transactionManager.submit(transaction, ethsigner)
.. they all work .. just wondering what's the nicest to read..
...thoughts only...

Reply:
I'd say, language wise ...you've got the relationship between objects backwards (in the western language context).

With two objects as nouns and the operations as a verb, the syntax is usually _noun_ _verb_ _noun_.
Doesn't the bigger _noun_ go on the left? 

e.g.
`ethSigner.execute(transaction)`